examples/demo.js

```diff
@@ -55,7 +55,7 @@ window.onload = function () {
   // UI for the overall demo:
   var ui = defaultHtmlSequencerUi(sequencer);
 
-  // find any `src` parameters in URL hash and attempt to source image from them and run the sequencer
+  // Find any `src` parameters in URL hash and attempt to source image from them and run the sequencer.
```

Comment:
what is them?

Reply:
I meant a developer can't understand pronouns. Be explicit and clear.
and is used twice. Form different sentences.
Statement is unclear to me.